docs/data/date-pickers/custom-components/Tabs.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { TextField, Box } from '@mui/material';
```

Comment:
```suggestion
import TextField from '@mui/materia/TextField';
import Box from '@mui/materia/Box';
```

We avoid root import from `@mui/material` 
We have some for utilities that are only exported from the root and it's not great because it bundles the whole package on dev (which is quite heavy).
But I think it's worth doing the deep imports on the doc whenever possible to give good habits to our users.